src/server.rs

```diff
@@ -12,7 +12,9 @@ use websocket::sync::Client;
 use websocket::{ClientBuilder, OwnedMessage, Message};
 use websocket::header::Headers;
 use std::net::TcpStream;
-use log::{debug, info, trace};
+use tracing::{info, debug, error, trace};
+use tracing_subscriber::fmt::format::FmtSpan;
+use tracing_subscriber::{EnvFilter, fmt};
```

Comment:
why use tracing_subscriber::fmt? looks unused. also these uses are duplicated on lines 30 and 31 for some reason